docker-compose.yml

```diff
@@ -62,6 +62,7 @@ services:
       DEFAULT_GRAPH: "http://mu.semte.ch/application"
     volumes:
       - ./data/db:/data
+      - ./config/virtuoso/virtuoso.ini:/data/virtuoso.ini # Note: on production override this setting
```

Comment:
As far as my thinking goes, the `docker-compose.yml` file should be the production version. The `.dev.yml` should include the version for development (mapping the same internal path means that the latest mapping wins).

Reply:
That's also fair! I mainly based this approach/config on the one of LMB and Loket, to maintain a bit of consistency.